test/recipes/80-test_ssl_new.t

```diff
@@ -78,7 +78,7 @@ my %skip = (
   "15-certstatus.conf" => $no_tls || $no_ocsp,
   "16-dtls-certstatus.conf" => $no_dtls || $no_ocsp,
   "18-dtls-renegotiate.conf" => $no_dtls,
-  "19-mac-then-encrypt.conf" => $no_pre_tls1_3
+  "19-mac-then-encrypt.conf" => $no_pre_tls1_3 || disabled("tls1_2")
```

Comment:
Take a look in 19-mac-then-encrypt.conf.in. My guess would be that the final 3 sub-tests use TLSv1.2 only ciphersuites. It would probably be better just to disable those three sub-tests in the no-tls1_2 case. If you do that then you'll probably also need to add this test to `%conf_dependent_tests` in this file.